### PR TITLE
docs(connector-github): add GitHub README.md (en-US)

### PR DESCRIPTION
**Pull request by darcyYe**
## Summary

Add GitHub README.md (en-US).

## Testing

UTs.

### CODE REVIEW

**Review by IceHe — APPROVED** [review]
Overall LGTM

File: packages/connector-github/README.md [hunk 1]
```diff
@@ -1,2 +1,35 @@
...
+## Set up the Logto GitHub connector settings
+
+Fill out the `clientId` and `clientSecret` field with **Client ID** and **Client Secret** you've got from OAuth app detail pages mentioned in the previous section.
+
+## Settings
+
+|     Name     |  Type  |
+|:------------:|:------:|
+|   clientId   | string |
+| clientSecret | string |
```
Comment (gao-sun) [inline packages/connector-github/README.md:35]: will it be better to provide a sample json here? seems this is not clear enough

Comment (darcyYe) [reply packages/connector-github/README.md:35]: I was thinking that if the user is adding connectors for the first time, the contents of README are shown on their left, and a sample JSON (written in config-template.json in all connector folders) is already filled in the text input box. If developers are trying to update the settings of connectors,  the configuration in use (definitely a valid JSON sample) is shown in the text input box. Maybe there is no need to add a sample JSON in readme files as developers can always find a valid JSON without effort?

Comment (darcyYe) [reply packages/connector-github/README.md:35]: `Settings` here is used as a reference for developers.

File: packages/connector-github/README.md [hunk 2]
```diff
@@ -1,2 +1,35 @@
...
+
+Go to [GitHub website](https://github.com/) and sign in with your GitHub account. You may register a new account if you don't have one.
+
+## Create and configure GitHub OAuth app
+
+Follow the [Creating an OAuth App](https://docs.github.com/en/developers/apps/building-oauth-apps/creating-an-oauth-app) guide, and register a new application.
+
+Name your new OAuth application in **Application name** and fill up **Homepage URL** of the app.
+You can leave **Application description** field blank and customize **Authorization callback URL** as `${your_logto_origin}/callback/github-universal`.
+We suggest not to check the box before **Enable Device Flow**, or users who sign in with GitHub on mobile devices must confirm the initial sign-in action in the GitHub app. Many GitHub users do not install the GitHub mobile app on their phones, which could block the sign-in flow. Please ignore our suggestion if you are expecting end-users to confirm their sign-in flow.
```
[the remark this answers is not in the document]
Comment (darcyYe) [reply packages/connector-github/README.md:16]: Updated.

[the remark this answers is not in the document]
Comment (darcyYe) [reply packages/connector-github/README.md:16]: Updated.

File: packages/connector-github/README.md [hunk 3]
```diff
@@ -1,2 +1,45 @@
...
+Here is an example of GitHub connector config JSON.
+
+```json
+{
+    "clientID": "<your-client-id>",
+    "clientSecret": "<your-client-secret>"
+}
+```
+
+## Settings
```
Comment (gao-sun) [inline packages/connector-github/README.md:40]:
```suggestion
### Config types
```

File: packages/connector-github/README.md [hunk 4]
```diff
@@ -1,2 +1,45 @@
...
+You can also enable end-users to sign in with GitHub accounts by [creating a GitHub app](https://docs.github.com/en/developers/apps/building-github-apps/creating-a-github-app).
+
+## Managing GitHub OAuth apps
+
+Go to the [OAuth Apps page](https://github.com/settings/developers) and you can add, edit or delete existing OAuth apps.
+You can also find `Client ID` and generate `Client secrets` in OAuth app detail pages.
+
+As for managing GitHub Apps we mentioned in the last section, go to the [GitHub Apps page](https://github.com/settings/apps).
+
+## Set up the Logto GitHub connector settings
```
Comment (gao-sun) [inline packages/connector-github/README.md:27]: I think we can align the format of this section. I'll add the types section in wechat native later

```suggestion
## Compose the connector JSON
```

**Review by gao-sun — APPROVED** [review]

**Pull request merged**